docs/network/edges.html

```diff
@@ -285,7 +285,9 @@
             <td class="indent">arrows.middle</td>
             <td>Object or Boolean</td>
             <td><code>Object</code></td>
-            <td>Exactly the same as the to object but with an arrowhead in the center of the edge.</td>
+            <td>In general, the same as the to object but with an arrowhead in the center of the edge.
+                The only difference is that the direction of the arrow can be flipped by using a negative
+                value for <code>arrows.middle.scaleFactor</code>.</td>
```

Comment:
Consider dropping superfluous wording and wrapping the word to in single quotes:
```
<td>Similar to the 'to' object, but with an arrowhead in the center of the edge. The direction of the arrow can be flipped by using a negative value for <code>arrows.middle.scaleFactor</code>.</td>
```